src/main/kotlin/com/autonomousapps/internal/Bundles.kt

```diff
@@ -144,5 +151,12 @@ internal class Bundles(private val dependencyUsages: Map<Coordinates, Set<Usage>
 
       return bundles
     }
+
+    private fun coordinatesOrPathEquals(coordinates: Coordinates, primaryId: String) =
+      coordinates.identifier == primaryId || (coordinates is IncludedBuildCoordinates) && coordinates.resolvedProject.identifier == primaryId
```

Comment:
the parentheses look redundant here. did you mean the `)` to include the third boolean as well, after the `&&`?

Reply:
It's redundant. Good catch. Removed.